notebook.ipynb

```diff
@@ -92,37 +92,1248 @@
+      "text/plain": [
+       "<Figure size 1800x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(25, 6))\n",
+    "scatter = ax.scatter(list(postalcode[\"ws\"]),\n",
+    "                list(postalcode[\"ssrd\"]),\n",
+    "                color=\"blue\")\n",
+    "\n",
+    "ax.set(title=\"Windspeed compared to the solar surface radiation downwards \\n\",\n",
```

Comment:
Add a semicolon at the end so no text appears above the graph.
Same goes for the rest of the graphs.

Reply:
The output text is gone